.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+  ],
+  root: true,
+  rules: {
+    "comma-dangle": "off",
+    "eqeqeq": "warn",
+    "no-console": "warn",
+    "no-unused-vars": ["error", {vars: "all", args: "none", ignoreRestSiblings: false}],
+    "no-var": "error",
+    "prefer-const": "error",
```

Comment:
Let's not do prefer-const please. I really hate seeing `for (const foo of bar)` for example. My brain is used to consts being literals for stuff like like cookie lifetime, etc.